flask/app/variants.py

```diff
@@ -67,7 +66,6 @@ def get_report_df(df: pd.DataFrame, type: str):
                 "variation": "first",
```

Comment:
chromosome is present in participant-wise report but not variant-wise
```suggestion
                "variation": "first",
                "chromosome": "first",
```